src/index.js

```diff
@@ -286,6 +285,7 @@ class WPTS {
 
         const options = seleniumAssistantBrowser.getSeleniumOptions();
         options.addArguments(`user-data-dir=${tempPreferenceFile}/`);
+        options.addArguments(`no-sandbox`);
```

Comment:
This is a bit weird because this tool does run on Travis but it needs XVFB to set up and running.

https://github.com/GoogleChrome/web-push-testing-service/blob/master/.travis.yml#L24

I would be more comfortable accepting if it was optional (Sorry to be a pain)